modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -472,8 +472,18 @@ function validateOptionalValues(shares: string[], start: number, end: number, sh
  */
 export function parseKShare(share: SignatureShareRecord): KShare {
   const shares = share.share.split(delimeter);
-  validateSharesLength(shares, 11, 'K');
+  // TODO: once p and sigma are mandatory, make the exepcted length 11 + 2 * EcdsaPaillierProof.m
```

Comment:
nit: Might want to include ticket BG-78794 or whatever here so we don't forget

Reply:
Once the types are updated, this function will cause a compiler error since we can no longer pass an undefined value :)